src/components/interaction-designer/clipboard/ClipboardRoot.vue

```diff
@@ -108,7 +144,8 @@ export default {
           return
         }
         const { prompt }: IRichCursorInputRequired = cursor
-        this.blockPrompts.push(prompt)
+        this.blocksData.push({ prompt, isFocused: true })
+        this.setBlocksData(this.blocksData)
```

Comment:
For future reference, I don't think we should need to do this, because vue should respond to `blocksData.push()`. Were you running into situations where things weren't reactive?